src/Features/CSharp/Portable/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs

```diff
@@ -252,7 +254,7 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
             }
 
             var declarator = variableDeclaration.Variables[0];
-            if (declarator.Initializer == null)
+            if (declarator.Initializer?.Value == null)
```

Comment:
❔ What impact did this change have specifically? Is it looking at expressions like this?

```
Action a = null;
```

If so, do we have tests covering this that depend on the new condition?

Reply:
undid this.